_opensearch/rest-api/tasks.md

```diff
@@ -49,7 +49,13 @@ Note that if a task finishes running, it won't be returned as part of your reque
           "start_time_in_millis": 1613599752458,
           "running_time_in_nanos": 994000,
           "cancellable": false,
-          "headers": {}
+          "headers": {},
+          "resource_stats" : {
+            "total" : {
+              "cpu_time_in_nanos" : 0,
+              "memory_in_bytes" : 0
+            }
```

Comment:
These fields will only be present if detailed flag is set and will not be included as a part of default response. This behaviour is similar to ```description``` field which is not present in this sample response and hence I think we should not include these fields as well.